go.mod

```diff
@@ -13,7 +13,7 @@ require (
 	github.com/hashicorp/go-memdb v1.3.2
 	github.com/hexops/gotextdiff v1.0.3
 	github.com/imdario/mergo v0.3.12
-	github.com/kong/go-kong v0.25.2-0.20220123181121-e965de22d701
+	github.com/kong/go-kong v0.25.2-0.20220126173454-9aab6cf4ade8
```

Comment:
Blocker for this PR.
Let's get the upstream dependent PR merged. And then do a release of go-kong and then bump it up here.
These commit hash-based dependencies are a bit scary since they can point to a stale commit.

Reply:
indeed

Reply:
I created a new go-kong release and updated the dependence here